metrics_utility/logger.py

```diff
@@ -5,6 +5,10 @@
 logging.basicConfig(format='%(message)s', level=logging.INFO)
 logger = logging.getLogger(__name__)
 
+# This logger will log all message info and up
+logger_info_level = logging.getLogger(__name__)
```

Comment:
Maybe please do not override our default config.

Reply:
This was discussed with @himdel , I need to overwritde the logger because whatever the log level is, I want to see that information in the log and so we can see the math for the total vcpu calculation and this in case a customer ask for bill proof.